include/sys/dkioc_free_util.h

```diff
@@ -48,7 +48,7 @@ typedef struct dkioc_free_list_s {
 } dkioc_free_list_t;
 
 static inline void dfl_free(dkioc_free_list_t *dfl) {
-	kmem_free(dfl, DFL_SZ(dfl->dfl_num_exts));
+	vmem_free(dfl, DFL_SZ(dfl->dfl_num_exts));
```

Comment:
Shouldn't there be a `dfl_alloc()` counterpart to prevent this kind of issue.  I see it's directly `vmem_alloc()` in the trim code.